templates/shared/_pro-contact-us-form.html

```diff
@@ -218,40 +246,26 @@ <h2 class="p-heading--3">What advice are you looking for?</h2>
           <h2 class="p-heading--3">About you</h2>
           <ul class="p-list">
             <li class="p-list__item">
-              <label for="firstName">First name:</label>
+              <label for="firstName">First name*:</label>
```

Comment:
There's a Vanilla class we should use for these instead https://vanillaframework.io/docs/base/forms#required